packages/addons/service/fd628/sources/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.fd628"
+		name="FD628 Display"
+		version="@PKG_VERSION@"
+		provider-name="LibreELEC">
```

Comment:
"Team LibreELEC" .. if we follow the same as other add-ons (and services, like service.slice)